src/lib/OpenEXRCore/parse_header.c

```diff
@@ -2073,6 +2073,7 @@ internal_exr_compute_chunk_offset_size (struct _internal_exr_part* curpart)
     const exr_attr_chlist_t* channels     = curpart->channels->chlist;
     uint64_t                 unpackedsize = 0;
     int64_t                  w;
```

Comment:
I kind of wonder if w should be uint64_t in the first place, rather than casting it a few times through the following code? Or is there another reason to keep it signed earlier in the computation?

Reply:
well, I kept it signed because the data window is signed, although that is not a great reason

Reply:
I forget if we allow negative data windows? I remember recently there was discussion around it, and I don't remember how it shook out.

Reply:
Yes, of course. That's the only way to have "overscan" pixels.

Reply:
Right, thanks!

Reply:
min.x and max.x are both allowed to be negative, but max.x cannot be less than min.x, so w could be unsigned.